src/v/rpc/types.cc

```diff
@@ -59,9 +59,20 @@ std::ostream& operator<<(std::ostream& o, const status& s) {
         return o << "rpc::status::request_timeout";
     case status::server_error:
         return o << "rpc::status::server_error";
+    case status::version_not_supported:
+        return o << "rpc::status::version_not_supported";
     default:
         return o << "rpc::status::unknown";
     }
 }
 
+std::ostream& operator<<(std::ostream& o, transport_version v) {
+    switch (v) {
+    case transport_version::v0:
+        return o << "rpc::transport_version::v0";
+    case transport_version::unsupported:
+        return o << "rpc::transport_version::unsupported";
```

Comment:
nit: might be more future-proof to have a default case that servers as an unsupported version, perhaps also casting the value of v

Reply:
Not sure why this double posted.